tsl/test/expected/compression_errors.out

```diff
@@ -321,23 +321,21 @@ alter table table_constr add constraint table_constr_fk_add_after FOREIGN KEY(d)
 ERROR:  operation not supported on hypertables that have compression enabled
 -- ddl ADD column variants that are not supported
 ALTER TABLE table_constr ADD COLUMN newcol integer CHECK ( newcol < 10 );
-ERROR:  cannot add column with constraints or defaults to a hypertable that has compression enabled
+ERROR:  cannot add column with constraints to a hypertable that has compression enabled
 ALTER TABLE table_constr ADD COLUMN newcol integer UNIQUE;
-ERROR:  cannot add column with constraints or defaults to a hypertable that has compression enabled
+ERROR:  cannot add column with constraints to a hypertable that has compression enabled
 ALTER TABLE table_constr ADD COLUMN newcol integer PRIMARY KEY;
-ERROR:  cannot add column with constraints or defaults to a hypertable that has compression enabled
+ERROR:  cannot add column with constraints to a hypertable that has compression enabled
```

Comment:
I do not see the old error triggered anywhere, but it is still in the code. Do you have a test case for that? I don't see one.